@here/harp-webpack-utils/scripts/HarpWebpackConfig.js

```diff
@@ -96,11 +110,14 @@ export function addHarpWebpackConfig(config?: Configuration, harpConfig?: HarpWe
     return bundles;
```

Comment:
Return bundle[0] if there is only one bundle, so you don't have to resort to
```
addHarpConfig(...)[0];
```
later.